packages/bot-web-ui/src/pages/server-side-bot/add-bot/form-wrappers/__tests__/mobile-form-wrapper.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import MobileFromWrapper from '../mobile-form-wrapper';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
+
+jest.mock('../../../../../xml/martingale.xml', () => '');
+
+jest.mock('@deriv/bot-skeleton', () => ({
+    ...jest.requireActual('@deriv/bot-skeleton'),
+    ApiHelpers: {
+        instance: {
+            contracts_for: {
+                getDurations: jest.fn(),
+                getMarketBySymbol: jest.fn(),
+                getSubmarketBySymbol: jest.fn(),
+                getTradeTypeCategoryByTradeType: jest.fn(),
+            },
+        },
+    },
+}));
+
+window.Blockly = {
+    Xml: {
+        textToDom: jest.fn(),
+        domToText: jest.fn(),
+    },
+};
+
+describe('<MobileFormWrapper />', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element, mock_DBot_store: RootStore | undefined;
+    const mock_onSubmit = jest.fn();
```

Comment:
Since some comments are not addressed yet, noticed one more nit-pick :)
```suggestion
    const mockOnSubmit = jest.fn();
```
and below: `mock_DBot_store` -> `mock_dbot_store`, same as I mentioned in `form.spec.tsx`

Reply:
This is fixed @maryia-deriv 🙇